ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -184,7 +184,11 @@ def _stop_service(host, service_name):
             serverid = (realm_to_serverid(host.domain.realm)).upper()
             service_name = 'dirsrv@%s.service' % serverid
         elif service_name == 'named':
-            service_name = 'named-pkcs11'
+            # The service name may differ depending on the host OS
+            script = ("from ipaplatform.services import knownservices; "
+                      "print(knownservices.named.systemd_name)")
+            result = host.run_command(['python3', '-c', script])
+            service_name = result.stdout_text.strip()
```

Comment:
This diff is taken from https://pagure.io/freeipa/c/50fdc8089a4f7970d6dfa3dc843b75b3e6f31d3c that was needed only on the master branch. Since this PR is against ipa-4-8, are you sure this is needed?

Reply:
The code has been removed.